### DISCUSSION TITLE
Release v0.9.8c / 5.53.2

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1332] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixes many bugs.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# Changelog

### Added
- added explicit lines on Plus installer to delete empty shell registry keys at uninstall time (by mpheath) [3f661a8](https://github.com/sandboxie-plus/Sandboxie/commit/3f661a8d49137b6d2c3e00757952c71b0df11e4d)

### Fixed
- fixed template sections not showing in editor [#1287](https://github.com/sandboxie-plus/Sandboxie/issues/1287)
- fixed autodelete box content broken in the previous build [#1296](https://github.com/sandboxie-plus/Sandboxie/issues/1296) [#1324](https://github.com/sandboxie-plus/Sandboxie/issues/1324)
- fixed crash in "Browse Content" window [#1313](https://github.com/sandboxie-plus/Sandboxie/issues/1313)
- fixed issue with icon resolution on Windows 7 [#1310](https://github.com/sandboxie-plus/Sandboxie/issues/1310)
- fixed invalid "No Inet" status in the status column [#1312](https://github.com/sandboxie-plus/Sandboxie/issues/1312)
- fixed Windows Explorer search box not working (by isaak654) [#1002](https://github.com/sandboxie-plus/Sandboxie/issues/1002)
- fixed Waterfox phishing template (by Dyras) [#1309](https://github.com/sandboxie-plus/Sandboxie/pull/1309)
- fixed issue with Chinese translation files on Plus installer (by mpheath) [#1317](https://github.com/sandboxie-plus/Sandboxie/issues/1317)
- fixed autorun registry key path on Plus installer (by mpheath) [abd2d44](https://github.com/sandboxie-plus/Sandboxie/commit/abd2d44cd6f305da956ad70c7481cb1256efff24)
- fixed memory corruption in SbieSvc.exe

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/0.9.8c'>Release v0.9.8c / 5.53.2</a>.</em>

**Comment by BilleBarrett** [comment]
Updated to v0.9.8c the only problem is that Edge Chrome open and noting shows on the desktop
I checked I did nothing to the INI file.
Will uninstall 8c and put 8b back and see if Chrome and Edge load,

**Reply by isaak654** [reply]
Please check also if disabling hardware acceleration improves the situation.

**Reply by BilleBarrett** [reply]
I had to remove MS Chrome and reinstall the Enterprise version and this seems to cure the problem :-)

**Comment by BilleBarrett** [comment]
Never turn it on learned that years ago with Sandboxie



 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Comment by BilleBarrett** [comment]
I uninstalled Sandboxie 7 Plus, clean up the ini files

Ran Repair on Microsoft Edge, 

Installed 5.53.2. 

Chrome is having no problem, I check all the points in Microsoft Edge except Force Edge

Edge Chrome runs as soon as I check the force box it fails
Edge Chrome Version 95.0.1020.30 (Official build) (64-bit)

Ok it looks like there is something screwed up with system, I just installed Sandboxie 5.53.2 on a clients laptop and check all the boxes and it ran ☹

  _____  

#

# Sandboxie-Plus configuration file

#

 

[GlobalSettings]

Template=WindowsRasMan

Template=WindowsLive

Template=KeyScrambler

Template=OfficeLicensing

Template=OfficeClickToRun

Template=RoboForm

FileRootPath=M:\Sandbox\%USER%\%SANDBOX%

 

[DefaultBox]

ConfigLevel=9

BlockNetworkFiles=y

Template=Edge_Profile_DirectAccess

Template=Edge_Phishing_DirectAccess

Template=Edge_Sync_DirectAccess

Template=Edge_Preferences_DirectAccess

Template=Edge_Passwords_DirectAccess

Template=Edge_Cookies_DirectAccess

Template=Edge_History_DirectAccess

Template=Edge_Bookmarks_DirectAccess

Template=Chrome_Force

Template=Firefox_Profile_DirectAccess

Template=Firefox_Session_DirectAccess

Template=Firefox_Passwords_DirectAccess

Template=Firefox_Cookies_DirectAccess

Template=Firefox_Bookmarks_DirectAccess

Template=Firefox_Force

Template=Office_Outlook

Template=nVidia_Stereoscopic3D

Template=nVidia_nView

Template=Chrome_Sync_DirectAccess

Template=Chrome_Preferences_DirectAccess

Template=Chrome_Passwords_DirectAccess

Template=Chrome_Cookies_DirectAccess

Template=Chrome_Bookmarks_DirectAccess

Template=Chrome_History_DirectAccess

Template=Chrome_Profile_DirectAccess

Template=AdobeAcrobatReader

Template=AdobeDistiller

Template=AdobeAcrobat

Template=OpenSmartCard

Template=OpenBluetooth

Template=SkipHook

Template=FileCopy

Template=qWave

Template=BlockPorts

Template=LingerPrograms

Template=Chrome_Phishing_DirectAccess

Template=Firefox_Phishing_DirectAccess

Template=AutoRecoverIgnore

RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%

RecoverFolder=%Personal%

RecoverFolder=%Desktop%

BorderColor=#00FF00,ttl,6

Enabled=y

BoxNameTitle=-

AutoRecover=y

AutoDelete=y

NeverDelete=n

ForceFolder=W:\

ForceProcess=acrord32.exe

CopyLimitKb=249152

 

[UserSettings_10FE026B]

SbieCtrl_UserName=billeb

SbieCtrl_BoxExpandedView=DefaultBox

SbieCtrl_NextUpdateCheck=1635797235

SbieCtrl_AutoApplySettings=y

SbieCtrl_SettingChangeNotify=n

SbieCtrl_WindowCoords=200,150,1237,632

SbieCtrl_ActiveView=40021

SbieCtrl_HideWindowNotify=n

 

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Comment by BilleBarrett** [comment]
Well it all boiled down to the fact that I had to totally uninstall and removed a reference to Chrome Edge using revo uninstalled 
After that I download the c version and Microsoft Chrome edge came back to life 
Now I update a few clients and they are bits error report and I do not know how to turn that off

Can you help tis is the Classic version

Bill e. Barrett

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Comment by akirasada** [comment]
hi, please check this 
https://youtu.be/6BIKuj21E0s
ms edge in sandboxie 5.53.2 windows 11 x64

**Comment by BilleBarrett** [comment]
I have it running on Plus latest version OK

Thank you

 

Barrett Computers [Remote Desktop Services]
517 Miltwood Dr,  
Greensboro, NC 27455-2709
(336) 988-0891
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
 <https://www.facebook.com/DesktopRemoteServices/> https://www.facebook.com/DesktopRemoteServices/

**Comment by bjm234** [comment]
Curious 1, why Run -> Command Prompt & Boxed Tools -> Command Prompt/s. 
What's difference? 
Edit: okay....Run -> Command Prompt is sans Administrator.  

Curious 2:  Language UI - American English -> Apply reverts to International English. 
Are International English and American English the same, at this time?
